.dotnet/src/Custom/Files/Internal/InternalUploadsClient.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+
+namespace OpenAI.Files;
+
+[CodeGenClient("Uploads")]
+internal partial class InternalUploadsClient
+{
+    protected internal InternalUploadsClient(ClientPipeline pipeline, Uri endpoint, OpenAIClientOptions options)
```

Comment:
nit because this is currently internal, but...

- You can delete this constructors; we don't have constructors that take the endpoint anymore.
- You can copy-paste the three custom constructors from the `FileClient` and just change the name to `InternalUploadsClient`. 
- You can suppress the generated constructor that takes the endpoint parameter by adding: 
`[CodeGenSuppress("InternalUploadsClient", typeof(ClientPipeline), typeof(ApiKeyCredential), typeof(Uri))]`